### ISSUE TITLE
Wrong Auto Indention of if/else/end, case/when/end

### ISSUE THREAD

**Issue by kassiansun** [bbatsov/rubocop#3507]
```ruby
class A
  def execute
    effective_days = if @data['form_last']
      @days.last(@data['effective_days'] || 1)
    else
      @days
    end
  end
end
```
--------

## Expected behavior
```
$ rubocop -a test.rb
```

```ruby
class A
  def execute
    effective_days = if @data['form_last']
                       @days.last(@data['effective_days'] || 1)
                     else
                       @days
                     end
  end
end
```
## Actual behavior
```ruby
class A
  def execute
    effective_days = if @data['form_last']
                       @days.last(@data['effective_days'] || 1)
                     else
                       @days
    end
  end
end
```
## Steps to reproduce the problem

```
$rubocop -a test.rb
```

## RuboCop version

Include the output of `rubocop -V`:

```
$ rubocop -V
0.42.0 (using Parser 2.3.1.3, running on ruby 2.3.1 x86_64-linux)
```

**Comment by bquorning** [comment]
This is because the default configuration for `Lint/EndAlignment` sets `AutoCorrect: false` – see #1789.

**Comment by kassiansun** [comment]
@bquorning Great MR. Actually I use editors for indention, which is a bit more robust, as it's only lexical.
